stream-chat-android-ui-components/src/main/res/values/colors.xml

```diff
@@ -74,4 +74,7 @@
 
     <!-- Message replies -->
     <color name="stream_ui_bars_background">#FFFFFF</color>
+
+    <!-- Make status bar icons dark -->
+    <bool name="stream_ui_light_status_bar">true</bool>
```

Comment:
I'm not the happiest about keeping a boolean in the colors res, but it does govern how colors behave, so I guess it makes sense